profile/utils.js

```diff
@@ -331,7 +331,7 @@ async function getSelfUser() {
 }
 
 async function getUser(userId) {
-  const userResponse = await fetch(`${API_BASE_URL}/users/userId/${userId}`, {
+  const userResponse = await fetch(`${API_BASE_URL}/users/?id=${userId}`, {
```

Comment:
We can remove the extra slash after `users`.

Reply:
Done